### ISSUE TITLE
[a11y] 9.1.3.1b HTML structural elements for lists

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10723]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-3-1b-html-strukturelemente-fuer-listen

### Personal, Shares, Spaces, deleted Files: Rather fulfilled.

- [ ] The list in the sidebar with the links such as "Personal" is incorrectly implemented. It is
not permitted to use other elements below "ul" elements instead of "li" elements
such as "div" elements.
![image](https://github.com/owncloud/web/assets/132444655/9d31786e-dc48-4c53-bd8f-5fd171ec9754)

Info: Account: Fulfilled

**Comment by pascalwengerter** [comment]
Which version are/were you testing? I can't reproduce the exact issue but would like to know whether the additional `div` within the `nav` tag remains a violation?

<img width="440" alt="Screenshot 2024-04-11 at 09 32 15" src="https://github.com/owncloud/web/assets/16822008/ca72ea35-9532-4b04-99f4-1a93f4407826">

**Comment by tschiebel** [comment]
@tbsbdr cant find this info - do you know that?

**Comment by pascalwengerter** [comment]
Never mind, I do have a rough timestamp of when the tests were being done by now ;) working on fixes

**Comment by pascalwengerter** [comment]
Tried again and can't replicate - not sure if the `nav-highligther` div should me moved out of the `<nav>` tag, but generally not the same level of a11y violation it was at the time this issue stems from

**Comment by kulmann** [comment]
Assessment is correct, the nav highlighter div doesn't live within the `<ul>` anymore (which would be an a11y violation), but is a child of the `<nav>` tag by now.

The nav highlighter div is aria-hidden and it's valid html to have a div inside a nav. We can close here, this is done.
